### PR TITLE
Reader bugfix

### PR DESCRIPTION
**Pull request by banorawhite**

### CODE REVIEW

File: app/Http/Controllers/ScanController.php [hunk 1]
```diff
@@ -32,21 +32,20 @@ public function store(Request $request) {
                                     ->where('date', $date_now)->first();
 
             if ($attendance && $attendance->time_in) {
-                Notification::send($user, new Overtime($employee));
```
Comment (banorawhite) [inline app/Http/Controllers/ScanController.php:35]: @kielboy8 need ba 'tong line na 'to?

File: app/Http/Controllers/ScanController.php [hunk 2]
```diff
@@ -32,21 +32,20 @@ public function store(Request $request) {
...
-                // if($totalTime > 32400) {
-                //     auth()->user()
-                // }
+                $totalTime = Attendance::selectRaw('(time_out - time_in) difference')->where('attendance_id', request('id'))->first()->difference;
+                if($totalTime > 32400) {
+                    Notification::send($user, new Overtime($employee));
+                }
             }
             else {
                 Notification::send($user, new Overtime($employee));
```
Comment (banorawhite) [inline app/Http/Controllers/ScanController.php:43]: @kielboy8 pati eto?

File: app/Http/Controllers/ScanController.php (hunk 1 above)
Comment (kielboy8) [reply app/Http/Controllers/ScanController.php:35]: yes kasi yan yung magsesend ng notif

File: app/Http/Controllers/ScanController.php (hunk 2 above)
Comment (kielboy8) [reply app/Http/Controllers/ScanController.php:43]: yes for overtime yan

**Pull request merged**